lib/completion.dart

```diff
@@ -63,8 +63,8 @@ class DartCompleter extends CodeCompleter {
 
       List<Completion> completions =  analysisCompletions.map((completion) {
         // TODO: Move to using a LabelProvider; decouple the data and rendering.
-        String displayString = completion.isMethod ? '${completion.text}()' : completion.text;
-        if (completion.isMethod && completion.returnType != null) {
+        String displayString = (completion.isMethod || completion.type == "METHOD") ? '${completion.text}()' : completion.text;
```

Comment:
Instead of changing this comparison (and the one below it), we should fix the `completion.isMethod` method.